.github/workflows/pr.yaml

```diff
@@ -123,6 +123,9 @@ jobs:
           composer --no-interaction --working-dir=drupal config repositories.drupal composer https://packages.drupal.org/8
 
           composer --no-interaction --working-dir=drupal config --no-plugins allow-plugins.cweagans/composer-patches true
+          composer --no-interaction --working-dir=drupal config --no-plugins allow-plugins.zaporylie/composer-drupal-optimizations true
+          composer --no-interaction --working-dir=drupal config --no-plugins allow-plugins.simplesamlphp/composer-module-installer true
+
```

Comment:
See https://github.com/OS2Forms/os2forms/blob/664589b8ece6f7ec9b11d4f6e23016f2ca3d6335/.github/workflows/pr.yml#L112 and https://github.com/OS2Forms/os2forms/blob/develop/scripts/code-analysis for a more clean way to do this.